src/python/tapkee.pyx

```diff
@@ -1,4 +1,2 @@
-#cython: language_level=3
```

Comment:
Failing attempt removing this line in case it would be the cause that the function doesn't appear in Python.